<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UrsaPQ Remote - Vacuum</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background-color: #FFFFFF;
}

QLabel {
	font-size: 12.5pt;
}

QGroupBox {
	background-color: #EFEFEF;
    border: 1px solid gray;
    border-color: #003959;
    border-radius: 10px;
}

QGroupBox::title {
    subcontrol-position: top left;
 	border-top-left-radius: 10px;
 	border-bottom-right-radius: 10px;
    padding: 5px 20px;
    background-color: #003959;
    color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>241</width>
      <height>286</height>
     </rect>
    </property>
    <property name="title">
     <string>Vacuum</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>46</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="prevacValveLockBox"/>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>136</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prevac valve Lock</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>46</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="pumpsEnableBox"/>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>45</y>
       <width>136</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pumps enable</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
